oceannavigator/frontend/src/components/DatasetSelector.jsx

```diff
@@ -370,7 +370,8 @@ class DatasetSelector extends React.Component {
     let quiverSelector = null;
     if (this.props.showQuiverSelector && !this.state.loading) {
       let quiverVariables = [];
-      if (this.state.datasetVariables) {
+      const ListvalidQuivModelClass = ["Mercator"]    // List of Valid Model_Class for Quiver Plot. Need to update this List when Model_Class : "Nemo" and "Fvcom" working
+      if ((ListvalidQuivModelClass.includes(this.state.model_class)) && (this.state.datasetVariables)) {
```

Comment:
Just a minor variable name tweak:

* Non-constant variables in JS should be defined using camelCase.

```suggestion
      const modelClassesWithQuiver = ["Mercator"]    // List of Valid Model_Class for Quiver Plot. Need to update this List when Model_Class : "Nemo" and "Fvcom" working
      if (this.state.datasetVariables && modelClassesWithQuiver.includes(this.state.model_class)) {
```

Also, `modelClassesWithQuiver` is a constant. I think this should be defined elsewhere in this file. Perhaps after `PARENT_ATTRIBUTES_TO_UPDATE`?

```js
const PARENT_ATTRIBUTES_TO_UPDATE = Object.freeze([
  "dataset",
  "dataset_attribution",
  "quantum",
  "variable",
  "variable_scale", // Default range values for variable
  "depth",
  "time",
  "starttime",
  "quiverVariable",
]);

const MODEL_CLASSES_WITH_QUIVER = Object.freeze(["Mercator"]);
```